packages/office-ui-fabric-react/src/components/Nav/Nav.styles.ts

```diff
@@ -191,27 +181,23 @@ export const getStyles = (props: INavStyleProps): INavStyles => {
           }
         }
       },
-      isGroup && [
-        {
-          width: '100%',
-          height: `${navHeight}px`,
-          borderBottom: `1px solid ${semanticColors.bodyDivider}`
-        },
-        DefaultFontStyles.large
-      ],
-      isLink && [
-        {
-          display: 'block',
-          width: `${leftPaddingExpanded - 2}px`,
-          height: `${navHeight - 2}px`,
-          position: 'absolute',
-          top: '1px',
-          left: `${position}px`,
-          zIndex: ZIndexes.Nav,
-          padding: 0,
-          margin: 0
-        }
-      ],
+      isGroup && {
+        fontSize: fonts.large.fontSize,
```

Comment:
For these, is there a reason we just change the font size and not the entire font variant? The font weight (regular) stays the same in this case between small and large, but what about for other cases where the larger font variant has a different font weight? Like xLarge's font weight of semibold. Is it all the other stuff that comes with the font variant that we just don't really need?

Reply:
Because we never go beyond `large` for this style section and like you mentioned everything stays the same so i only need to change the size.